src/main/java/com/sap/sgs/phosphor/fosstars/data/github/GitHubDataFetcher.java

```diff
@@ -405,27 +395,28 @@ protected Date expiration() {
    *
    * @throws IOException If something went wrong.
    */
-  protected void storeLocalRepositoriesInfo() throws IOException {
-    synchronized (LOCAL_REPOSITORIES_INFO_LOCK) {
+  protected static void storeLocalRepositoriesInfo() throws IOException {
```

Comment:
```suggestion
  private static void storeLocalRepositoriesInfo() throws IOException {
```

Or, package-private.